lib/utils/candidate.ts

```diff
@@ -1,13 +1,15 @@
 /**
  * The WebRTC API's [RTCIceCandidateStats](https://developer.mozilla.org/en-US/docs/Web/API/RTCIceCandidateStats)
  * dictionary which provides information related to an ICE candidate.
+ * @internalapi
```

Comment:
`@internalapi` means we don't show this from the "globals" page, but still accessible in the API docs page. For example, in our [globals page](https://twilio.github.io/rtc-diagnostics/globals.html), we only show the top level classes and methods. But if you open each class, you'll see the API definitions for other interfaces.